pyRMSD/test/testRMSDCalculators.py

```diff
@@ -249,6 +249,32 @@ def test_fitting_symmetry(self):
                                                                              ])
         rmsds = calculator.oneVsFollowing(0)
         numpy.testing.assert_almost_equal(expected_rmsds, rmsds, 12)
+        
+        
+    def test_fitting_symmetry_new(self):
```

Comment:
Test should have a docstring to describe what it's trying to test.

Reply:
Added docstrings in the two tests